wolfssl/internal.h

```diff
@@ -2153,6 +2155,24 @@ WOLFSSL_LOCAL int    TLSX_SNI_GetFromBuffer(const byte* buffer, word32 bufferSz,
 
 #endif /* HAVE_SNI */
 
+/* Trusted CA Key Indication - RFC 6066 (section 6) */
+#ifdef HAVE_TRUSTED_CA
+
+typedef struct TCA {
+    byte                       type;    /* TCA Type            */
+    byte*                      id;      /* TCA identifier      */
+    word16                     idSz;    /* TCA identifier size */
+    struct TCA*                next;    /* List Behavior       */
+#ifndef NO_WOLFSSL_CLIENT
+    byte                       options; /* Behavior options    */
```

Comment:
I do not see any use of `options` TCA struct member.

Reply:
Removed.